code/game/turfs/simulated/floor/plating/asteroid.dm

```diff
@@ -68,7 +68,27 @@
 			if(W.use_tool(src, user, 40, volume=50))
 				if(!can_dig(user))
 					return TRUE
-				to_chat(user, "<span class='notice'>You dig a hole.</span>")
+
+				if(W.tool_behaviour == TOOL_MINING && rand(1,15) == 1) //this may be too often - in testing 20 felt like too rare to be worth exploring
```

Comment:
`prob()` is better used here than rand, I think prob(7) gives roughly the same odds as what you've put

Reply:
oh thanks, that sounds cleaner too. will fix in a bit

Reply:
not sure I used prob right, testin g it now then gonna figure out how to correct it here lol